### PR TITLE
Covid tracking states fix

### PR DESCRIPTION
**Pull request by wh1210**

### CODE REVIEW

File: schema/dcschema_enum_classes.mcf [hunk 1]
```diff
@@ -383,3 +383,8 @@ typeOf: schema:Class
 subClassOf: schema:Enumeration
 name: "MedicalStatusEnum"
 description: "Medical status of a medical incident, indicating whether the patient is hospitalized, tested, deceased, etc."
+
+Node: dcid:MedicalTestResultTypeEnum
```
Comment (tjann) [inline schema/dcschema_enum_classes.mcf:387]: Can probably get rid of "Type" (note that this does not need data reimport. I'm also ok if you don't reimport the schema to dev.)

File: scripts/covid_tracking_project/historic_state_data/covid_tracking_new_schema.mcf [hunk 2]
```diff
@@ -0,0 +1,45 @@
+Node: dcid:CovidTrackingProject
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_state_data/covid_tracking_new_schema.mcf:1]: Can you put this node in the new file? and typeOf: StatisticSourceEnum

File: scripts/covid_tracking_project/historic_state_data/covid_tracking_new_schema.mcf [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+name: "MedicalTestResultTypeEnum"
+
+Node: dcid:Positive
+typeOf: dcs:MedicalTestResultTypeEnum
+name: "Positive"
+
+Node: dcid:Negative
+typeOf: dcs:MedicalTestResultTypeEnum
+name: "Negative"
+
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_state_data/covid_tracking_new_schema.mcf:24]: Are all of these duplicates?

Comment (wh1210) [reply scripts/covid_tracking_project/historic_state_data/covid_tracking_new_schema.mcf:24]: Yes, these are duplicated~

File: scripts/covid_tracking_project/historic_state_data/covid_tracking_new_schema.mcf (hunk 2 above)
Comment (wh1210) [reply scripts/covid_tracking_project/historic_state_data/covid_tracking_new_schema.mcf:1]: Yep, actually I already put this into enum_instances.mcf file. I will make the corresponding change.

File: schema/dcschema_enum_classes.mcf (hunk 1 above)
Comment (wh1210) [reply schema/dcschema_enum_classes.mcf:387]: Will do it :)

**Pull request merged**